kt/src/main/kotlin/org/example/Threads.kt

```diff
@@ -0,0 +1,108 @@
+package org.example
+
+import java.util.UUID.randomUUID
+import java.util.concurrent.Callable
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit
+import kotlin.concurrent.thread
+
+fun example1() {
+    val thread1 = Thread(runnable)
+
+    log("run new thread from main thread")
+    thread1.start()
+
+    `thread sleep`()
+
+    log("wait new thread")
+    thread1.join()
+
+    log("run new second thread from main thread")
+    val thread2 = thread(start = true, isDaemon = false, name = "handy-kt-thread") {
+        `thread started`()
+        `thread sleep`(1000)
+        `thread finished`()
+    }
+
+    log("wait new second thread")
+    thread2.join()
+    `thread finished`()
+}
+
+fun example2() {
+//    val executor = ThreadPoolExecutor(?, ?, ?)
```

Comment:
Remove comment?

Reply:
same as for `// val list = mutableListOf<Int>()`